test/banners/mobile/components/BannerCtrl.spec.ts

```diff
@@ -128,6 +132,16 @@ describe( 'BannerCtrl.vue', () => {
 		} );
 	} );
 
+	describe( 'Soft Close Submit Tracking', () => {
```

Comment:
We made the decision to leave the tests in there with skips when we prepared the pre-campaign tests because it's easier to figure out what to put back in when we re-add the features.